week3/homework/src/index.js

```diff
@@ -1,3 +1,197 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const express = require('express');
+const fs = require('fs');
+const bodyParser = require('body-parser');
+
+const app = express();
+
+app.use(express.json());
+
+app.use(bodyParser.json());
+app.use(express.urlencoded({ extended: false }));
+
+// Read All Todos
+app.get('/todos', (req, res) => {
+  const db = require('./db');
```

Comment:
It is much preferred to use fs.readFile - https://goenning.net/2016/04/14/stop-reading-json-files-with-require/